Unit4/Unit4-Project/README.md

```diff
@@ -0,0 +1,67 @@
+# Unit 4 Project: Creating a ToDo List App
+
+##### Credit to Shift_Up coach Noah Passalacqua
+![Become](https://avatars2.githubusercontent.com/u/38302861?s=200&v=4)
+
+### Welcome to the Unit 4 Project!
+
+Please read the instructions carefully and submit your project files to in the directory `Unit4/Unit4-Project`.
+
+Be creative when completing this project and add your own style. However, remember to stay focused on the main bullet points.
+
+##### Unit Project Instructions
+
+1. Create an `html`, `css`, and `js` file linking them together.
+
+1. Add an input field for users to add a todo item.
+
+1. Create an empty list element in the `<body>`.
+
+1. Create an event listener on the input field. On `ENTER`, it should append an item to the list with the user's text.
+
+1. For each item created in the list:
+    1. Add a checkbox to mark the item as complete. If checked, apply the ~~strikethrough~~ style to the item's text.
+    1. Add a button that will delete the item from the list.
+
+1. Create an empty state view for the list if there are no items in the list.
```

Comment:
Is the term "empty state view" familiar based on the OC material? Might be helpful to add a link to this term/concept in the OC course to help students refresh their memory. Im actually unclear myself exactly what you mean by this.